Gopkg.toml

```
@@ -166,3 +166,7 @@ ignored = [
     name = "k8s.io/kube-openapi"
     unused-packages = false
     non-go = false
+
+[[constraint]]
+  branch = "master"
```

Use a commit hash instead of the `master` branch.
Also, move this constraint block to line 141, where are the rest `constraints`.
Do not forget to run `make revendor`.
```suggestion
  revision = "a5d413f7728c81fb97d96a2b722368945f651e78"
```